crates/iota-json-rpc/src/indexer_api.rs

```diff
@@ -399,10 +400,13 @@ impl<R: ReadApiServer> IndexerApiServer for IndexerApi<R> {
                 .state
                 .get_dynamic_field_object_id(parent_object_id, name_type, &name_bcs_value)
                 .map_err(Error::from)?;
+
+            let options = options.unwrap_or(IotaObjectDataOptions::full_content());
+
             // TODO(chris): add options to `get_dynamic_field_object` API as well
```

Comment:
This can be be removed now

Reply:
We are passing `Option<IotaObjectDataOptions>` and this would just handle the case when `None` is passed. We could however refer to the default impl instead of `IotaObjectDataOptions::full_content()`. Adjusted it.

Reply:
I actually meant the TODO comment :sweat_smile:

Reply:
Changed it back to avoid any breaking changes if already used, and removed the comment as you suggested. https://github.com/iotaledger/iota/pull/6095/commits/97b106214cfd6a71ce4e7e64d22716ba2b0b0e02